.clang-format

```diff
@@ -1,13 +1,16 @@
 ﻿---
 BasedOnStyle: Google
-AlignAfterOpenBracket: AlwaysBreak
+AlignAfterOpenBracket: Align
+AllowAllParametersOfDeclarationOnNextLine: 'false'
 AlignConsecutiveAssignments: 'true'
 AlignEscapedNewlinesLeft: 'false'
 AllowShortBlocksOnASingleLine: 'false'
 AllowShortCaseLabelsOnASingleLine: 'false'
 AllowShortFunctionsOnASingleLine: Empty
 AllowShortIfStatementsOnASingleLine: 'false'
 AllowShortLoopsOnASingleLine: 'false'
+BinPackArguments: 'true'
+BinPackParameters: 'true'
```

Comment:
I'm not a huge fan of these options. In general, I think that if a function has so many arguments or parameters that they can't all fit on a single line, then each should get their own (for readabilities sake). That said, I don't have a super strong opinion on this, in general I'm not fussed about formatting as long as it's consistent.

Reply:
You do have a fair point; I prefer this because having one argument per line feels like a waste of space. 
It's just that linebreak after the opening parenthesis that bugs me the most.